client/common/sso_mib_tokens.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Copyright 2025 Siemens
+ */
+
+#ifndef FREERDP_CLIENT_COMMON_SSO_MIB_TOKENS_H
+#define FREERDP_CLIENT_COMMON_SSO_MIB_TOKENS_H
+
+#include <freerdp/freerdp.h>
+
+BOOL sso_mib_get_avd_access_token(freerdp* instance, char** token);
```

Comment:
this can be integrated in 2 ways to be available by default:
1. Move the whole thing to `libfreerdp/core` and check if a token can be retrieved before calling `instance->GetAccessToken(instance, ACCESS_TOKEN_TYPE_XXX`
2. Chain the `GetAccessToken` calls and expose a new callback via `rdpClientContext` and do not touch the existing callback in client implementations.